packages/dbos-cloud/login.ts

```diff
@@ -75,7 +76,14 @@ export async function authenticate(logger: Logger): Promise<string | null> {
   if (!deviceCodeResponse) {
     return null;
   }
-  console.log(`Login URL: ${deviceCodeResponse.verification_uri_complete}`);
+
+  const loginURL = deviceCodeResponse.verification_uri_complete
+  try {
```

Comment:
Confirmed that launching the browser here causes two browser windows to open when logging in via the extension. Can we comment out the `open` call for now until I get the extension moved over to using the DBOS Cloud API instead of the CLI?